### PR TITLE
EOS-19035: Problem: [Mini Provisioner] Handle path change for keys present under 'durability'

### PR DESCRIPTION
**Pull request by SwapnilGaonkar7**
Solution: Modified path to access correct path for data, parity and spare

### CODE REVIEW

**Comment by suvratjoshi** [comment]
Should also the change expected in `provisioning/miniprov/samples/sample_provisioner_cluster.json`
Line 55
```
          "durability":{
            "data": "1",
            "parity": "0",
            "spare": "0"
          }
```

File: provisioning/miniprov/hare_mp/templates/hare.config.conf.tmpl.json [hunk 1]
```diff
@@ -6,11 +6,9 @@
       },
       "storage_set": [
         {
-          "durability": {
-            "data": "1",
-            "parity": "0",
-            "spare": "0"
-          },
+          "data": "1",
```
Comment (knekrasov) [inline provisioning/miniprov/hare_mp/templates/hare.config.conf.tmpl.json:9]: @SwapnilGaonkar7 Please update create-template.sh script accordingly.

Comment (SwapnilGaonkar7) [reply provisioning/miniprov/hare_mp/templates/hare.config.conf.tmpl.json:9]: @knekrasov You mean this provisioning/miniprov/create-template.sh?

**Comment by SwapnilGaonkar7** [comment]
> Should also the change expected in `provisioning/miniprov/samples/sample_provisioner_cluster.json` ?
> Line 55
> 
> ```
>           "durability":{
>             "data": "1",
>             "parity": "0",
>             "spare": "0"
>           }
> ```

Done

File: provisioning/miniprov/hare_mp/templates/hare.config.conf.tmpl.json (hunk 1 above)
Comment (knekrasov) [reply provisioning/miniprov/hare_mp/templates/hare.config.conf.tmpl.json:9]: @SwapnilGaonkar7 Yes, that one.

**Review by knekrasov — APPROVED** [review]

**Review by suvratjoshi — APPROVED** [review]

**Pull request merged**